protoc-gen-openapiv2/main.go

```diff
@@ -37,6 +37,7 @@ var (
 	generateUnboundMethods     = flag.Bool("generate_unbound_methods", false, "generate swagger metadata even for RPC methods that have no HttpRule annotation")
 	recursiveDepth             = flag.Int("recursive-depth", 1000, "maximum recursion count allowed for a field type")
 	omitEnumDefaultValue       = flag.Bool("omit_enum_default_value", false, "if set, omit default enum value")
+	outputFormat               = flag.String("output_format", string(genopenapi.FormatJSON), "output content format")
```

Comment:
Can you list the valid formats in the description? Also, please also add this option to the bazel rules in the same directory.

Reply:
Added. Also added to bazel rules, but have not tested it yet.